terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -311,6 +311,8 @@ def remove_inconsequential_statements(statements: list[JSON]) -> list[JSON]:
 }
 
 clamav_image = 'clamav/clamav:0.104'
+dind_image = 'docker:19.03.15-dind'
+one_month = f'{30 * 24}h'  # One month in hours
```

Comment:
Only one reference and leaky abstraction (ask in PL if you want to know what I mean), so no point in extracting a variable. Also. don't encode the value of a variable in its name. I think I flagged this in previous PRs. It contradicts the point of a variable. Imagine a variable with hundreds of references. You don't really want to touch them all when you change the value.